Terminal-iOS/MyStudy/StudyDetail/ViewComponent/StudyDetail/StudyDetailView.swift

```diff
@@ -255,7 +255,7 @@ class StudyDetailView: UIViewController {
     }
     @objc func joinButtonDidTap() {
         TerminalAlertMessage.show(type: .StudyApplyView)
-        (TerminalAlertMessage.alertView as! TerminalAlertUIView).completeButton.addTarget(self, action: #selector(studyApplyMessageEndEditing), for: .touchUpInside)
+        (TerminalAlertMessage.alertView as! AlertBaseUIView).completeButton.addTarget(self, action: #selector(studyApplyMessageEndEditing), for: .touchUpInside)
```

Comment:
```suggestion
        (TerminalAlertMessage.alertView as! AlertBaseUIView).completeButton.addTarget(self, action: #selector(studyApplyMessageEndEditing), for: .touchUpInside)
```

이부분도 위와 같이 옵셔널 바인딩 처리를 통해서 하면 어떨까요??

Reply:
요거 나중에 적용해보겠슴다!